### ISSUE TITLE
The turbo process will not end when the terminal is shut down

### ISSUE THREAD

**Issue by cscxj** [vercel/turborepo#1626]
### What version of Turborepo are you using?

1.3.3

### What package manager are you using / does the bug impact?

pnpm

### What operating system are you using?

Mac

### Describe the Bug

I use turbo to run a development service. When I close vscode or the terminal, the development service does not stop and is still occupying the port.

### Expected Behavior

Automatically end the node process

### To Reproduce

You know

**Comment by sppatel** [comment]
Related - Ctr-C does not immediatley / gracefully shut down either - especially when turbo is running multiple task targets.  I usually have to resort to killing the pid.

**Comment by dobesv** [comment]
I have noticed a similar issue today, I think it's something that changed in the last version or two as this used to work.

**Comment by mehulkar** [comment]
I think this is the same issue as #908

**Comment by glinda93** [comment]
> Related - Ctr-C does not immediatley / gracefully shut down either - especially when turbo is running multiple task targets. I usually have to resort to killing the pid.

Still reproducible with the following environment

Turborepo Version: 1.5.5
Package Manager: pnpm 
Operating System: Windows 11 + Git Bash

**Comment by mehulkar** [comment]
@glinda93 mind leaving your comment on #908 instead?

**Comment by Srajan-Bansal** [comment]
> Related - Ctr-C does not immediatley / gracefully shut down either - especially when turbo is running multiple task targets. I usually have to resort to killing the pid.

I am having the same issue did you find any solution for it?

**Comment by Reyemneirda** [comment]
I am having the same issue since this morning

**Comment by lucaslosi** [comment]
same here. kicked in today

**Comment by florisdegraaff** [comment]
I too am having this problem, started this morning

**Comment by AnkanBis** [comment]
me too dude.  facing the same port occupied problem after ctrl + c the turborepo process. Any solution to fix ?

**Comment by inksnowhailong** [comment]
me too

**Comment by kotiyalashwin** [comment]
The only solution I was able to find was to manually diable the service running on the used port. In cmd write -> netstat -aon.
A list of processes will show, find 0.0.0.0:[port number]. Copy the PID. Then in task manager paste this PID and click End Task. Hope this works for you.

**Comment by Amith-AG** [comment]
> The only solution I was able to find was to manually disable the service running on the used port. In cmd write -> netstat -aon. A list of processes will show, find 0.0.0.0:[port number]. Copy the PID. Then in task manager paste this PID and click End Task. Hope this works for you.

I agree. My way is to run each app individually. This issue should remain open and should be resolved quickly, as it is causing inconvenience in development.

**Comment by tim-at-erdee** [comment]
This is definitely still an issue, for me in VSCode Ctrl+C'ing Turbo dev never stops the related processes. It is pretty annoying working from a dev container, as the ports get blocked by the old processes forcing me to restart.
Should a separate issue be created, since this one is closed?
